### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][Eclipse][wipro]:Unable to resize the columns in "Create Authentication files window"  for Azure signin automatically

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1368]
**Repro Steps:**
1. Open and Start Eclipse.
2. Open your project with Eclipse.
3. Press "Alt+T" to open  Tools menu, Navigate to  "Azure", and then select "Azure Sign In".
4. In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. After sign In Create Authentication Files window will opened .
7. In "Create Authentication files window" tab to subscription and try to resize the column's

**Expected Result:**
Keyboard users should able to do resize by using keyboard

**Actual Result:**
Keyboard user's unable to do resize the columns by using keyboard 

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![unable to resize columns by using keyboard in create authentication files window](https://user-images.githubusercontent.com/22956130/35680306-f6bcf8ba-077f-11e8-90f3-9865f9b5cae4.jpg)

**Comment by SummerSun** [comment]
The behavior is the same with eclipse table's behavior. User could use keyboard to resize the dialog, but not the table columns.

![image](https://user-images.githubusercontent.com/5179063/38789496-29164e3e-416d-11e8-8e9e-1a41449a0390.png)

**Comment by Eskibear** [comment]
external issue: 
Ref:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=308107

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749822
